src/Uno.Foundation/AsyncOperation.TResult.cs

```diff
@@ -64,20 +75,12 @@ public AsyncStatus Status
 		}
 
 		public void Cancel()
-		{
-			_cts.Cancel();
-		}
+			=> _cts.Cancel();
 
 		public void Close()
-		{
-			_cts.Cancel();
-		}
+			=> _cts.Cancel();
 
 		public TResult GetResults()
-		{
-			return _task.Result;
-		}
-
-		public Task<TResult> Task => _task;
+			=> Task.Result;
```

Comment:
Codacy found an issue: [Replace this use of 'Task.Result' with 'await'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)